WolvenKit.Common/Model/Arguments/ExportArgs.cs

```diff
@@ -301,6 +303,17 @@ public class MeshExportArgs : ExportArgs
         /// </summary>
         /// <returns>String</returns>
         public override string ToString() => $"GLTF/GLB | Export Type : {meshExportType} | Lod filter : {LodFilter} | Is Binary : {isGLBinary}";
+
+        public override ImportExportArgs AdjustDefaultFromFilepath(string baseFile)
+        {
+            // We don't want LOD filters to be active by default for weapon meshes 
+            if (baseFile.Contains("weapon", StringComparison.CurrentCultureIgnoreCase))
```

Comment:
I think this is too dangerous and it's also hidden. 

If we do this it should 
1. be behind a toggle switch in the UI
2. needs some description in the UI
3. needs a warning in the UI that this is active

Reply:
Could absolutely implement it as abstract but I didn't want to change 28 inheriting classes without touching base first :D That would make it not-hidden, at least.

And yeah, I noticed in hindsight that this doesn't only affect the default settings (or does it? This should only become active once, when the object is created…)